bin/plugin/commands/performance.js

```diff
@@ -147,13 +149,13 @@ async function getPerformanceResultsForBranch(
 	const results = [];
 	for ( let i = 0; i < 3; i++ ) {
 		await runShellScript(
-			'npm run test-performance',
+			`npm run test-performance -- packages/e2e-tests/specs/performance/${ testSuite }.performance.test.js`,
```

Comment:
Since there's already a "performance" folder, it seems `.performance` is redundant. (same for the results)

Reply:
a39f46d168a855ead81c5af2f0f0cb1e3d8eab71